manifests/0000_03_marketplace-operator_02_operatorhub.cr.yaml

```diff
@@ -0,0 +1,13 @@
+# This name of this file must appear after that of the file containing the OperatorHub CRD when ordered lexicographically.
+# The OperatorHub CRD file is sourced from the following vendor directory: ./vendor/github.com/openshift/api/config/v1
+apiVersion: config.openshift.io/v1
+kind: OperatorHub
+metadata:
+  name: cluster
```

Comment:
I confirmed that CVO won't have a problem with the (temporary) duplicate in openshift/cluster-config-operator. The PR to remove that is also queued up for after this merges: openshift/cluster-config-operator#245